docker/nginx/redirects-307.map

```diff
@@ -1,4 +1,5 @@
 ~^/mesosphere/dcos/latest/(.*)$ /mesosphere/dcos/2.0/$1;
+~^/ksphere/konvoy/latest/(.*)$ /ksphere/konvoy/1.5/$1;
```

Comment:
It looks like we might need more than just this for the redirects to work properly. I tried to substitute URL with `latest` and the left navbar loses the versions and it seems to point back to 1.4 actually, see screenshot 

<img width="681" alt="Screen Shot 2020-04-16 at 3 12 27 PM" src="https://user-images.githubusercontent.com/4225547/79511915-1d174200-7ff5-11ea-82a9-cbc7104e1382.png">

I saw this both on your hosted site and my local build

Reply:
Another thing is the left navbar will show `-beta` in the dropdown but not the main top left display so when looking through different pages you won't immediately recognize that these are beta docs 

<img width="529" alt="Screen Shot 2020-04-16 at 3 17 13 PM" src="https://user-images.githubusercontent.com/4225547/79512180-be05fd00-7ff5-11ea-92e4-055c0892efc1.png">